google/cloud/pubsub/samples/samples.cc

```diff
@@ -164,6 +187,9 @@ void AutoRun(std::vector<std::string> const& argv) {
   std::cout << "\nRunning CreateTopic() sample\n";
   CreateTopic(publisher_client, {project_id, topic_id});
 
+  std::cout << "\nRunning the StatusOr example" << std::endl;
```

Comment:
Choose between `std::endl` and `\n` (as is used in the other "Running ..." output).

Reply:
Done